mos_tests/rabbitmq_oslo/test_restarts.py

```diff
@@ -236,6 +235,32 @@ def consume_msg(remote, cfg_file_path):
     num_of_msg_consumed = int(re.findall('\d+', out_consume)[0])
     return num_of_msg_consumed
 
+
+def rabbit_rpc_server_start(remote, cfg_file_path):
+    logger.debug('Start [oslo_msg_check_server] on %s.' % remote.host)
+    cmd = 'oslo_msg_check_server --nodebug --config-file {0}'.format(
+        cfg_file_path)
+    remote.execute_async(cmd)
+
+
+def rabbit_rpc_client_start(remote, cfg_file_path):
+    logger.debug('Start [oslo_msg_check_client] on %s.' % remote.host)
+    cmd = 'oslo_msg_check_client --nodebug --config-file {0}'.format(
+        cfg_file_path)
+    remote.execute_async(cmd)
+    return remote.host
+
+
+def rabbit_rpc_curl(remote, host, port=5000):
+    # curl to client
+    cmd = 'curl -w %{http_code} -s -o /dev/null http://{host}:{port}'.format(
```

Comment:
`'curl -w %{{http_code}}'.format()` will return `'curl -w %{http_code}'`.
Is it what you want?